### ISSUE TITLE
🐛 [BUG] - Exit button for details item not working

### ISSUE THREAD

**Issue by ochu761** [SE310-Team1/ShoppIt#86]
### Description

Not working after [beabe28](https://github.com/SE310-Team1/ShoppIt/commit/beabe28e46936f94c918e01cfb9d494a229d4a7d)

Potential cause: hovering mouse over button does not show hover effect, perhaps button is covered by another FXML element?

### Reproduction URL

See commit [beabe28](https://github.com/SE310-Team1/ShoppIt/commit/beabe28e46936f94c918e01cfb9d494a229d4a7d)

### Reproduction steps

```bash
1. Goto search scene
2. Click on an item
3. Attempt to click on the "Exit" button
```

### Screenshots

```bash
![DESCRIPTION](LINK.png)
```

**Comment by ktha423** [comment]
This is very important issue, being able to exit the pop up is a good bug to tackle

**Comment by ochu761** [comment]
Closed by [1ed563a](https://github.com/SE310-Team1/ShoppIt/commit/1ed563a629731ce1c3723417ad6acf8f4cd355cb)
